Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -55,13 +57,21 @@ void Update()
         } 
     }
 
+    public void Drop()
+    {
+        rb.AddForce(GameManager.Instance.GetPlayerTransform().forward * thrustForce, ForceMode.Impulse);
+        hasBeenBumped = true;
+    }
+
     private void OnCollisionEnter(Collision other)
     {
         if (!hasBeenBumped || hasHit || other.gameObject.CompareTag("Player")) return;
 
         noisePing.SpawnNoisePing(other);
         hasHit = true;
 
+        UIManager.Instance.staminaBar.InstantIncreaseAwarenessBy(awakenessIncreasePercentage);

```

Comment:
AWARENESS OR AWAKENESS!! WE MUST CHOOSE ONE

Reply:
We should just name it awakareness

Reply:
this line is the epitome of the issue
u are right
😿